alacritty/src/display/mod.rs

```diff
@@ -622,6 +553,37 @@ impl Display {
         })
     }
 
+    #[inline]
+    pub fn gl_context(&self) -> &PossiblyCurrentContext {
+        self.context.get()
+    }
+
+    pub fn make_not_current(&mut self) {
+        if self.context.get().is_current() {
```

Comment:
What's the benefit of checking if the context is current here? Do things explode when trying to make a not-current context not-current, or is this just an efficiency thing?

Reply:
We just did that in the past, I'd assume that it could be faster?